sections/header.liquid

```diff
@@ -344,7 +344,7 @@
         <a href="{{ routes.root_url }}" class="header__heading-link link link--text focus-inset">
           {%- if section.settings.logo != blank -%}
             {%- assign image_size = section.settings.logo_width -%}
-            {%- assign image_size_2x = section.settings.logo_width | times: 2 -%}
+            {%- assign image_size_2x = image_size | times: 2 | at_most: 5760 -%}
```

Comment:
Added changes to the footer and header by reusing the assigned `image_size` variable instead of specifying `block.settings.image_width` and `section.settings.logo_width` (respectively) more than once.